lib/flashcard_runner.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'card'
+require_relative 'deck'
+require_relative 'turn'
+require_relative 'round'
+
+card_1 = Card.new("Where did Go-Go music originate?", "Washington DC", :Culture)
+card_2 = Card.new("Mass x Velocity equals what?", "Momentum", :STEM)
+card_3 = Card.new("What is the name of Washington DC's soccer team?", "DC United", :Culture)
+card_4 = Card.new("What is a Hokie?", "Nobody knows", :Culture)
+card_5 = Card.new("What river divides Washington DC from Virginia?", "Potomac", :Geography)
+cards = [card_1, card_2, card_3, card_4, card_5]
+deck = Deck.new(cards)
+round = Round.new(deck)
+
+def start(round)
+  count = 1
+  puts "Welcome! You're playing with #{round.deck.count} cards."
+  puts '-------------------------------------------------'
+  while count < round.deck.count
```

Comment:
Because this checks if `count < round.deck.count` it always ends the game one card short of the final card. You did correct it in your `flashcard_runner1.rb` file so I believe you can identify the issue.